subprojects/docs/src/docs/userguide/publishing_maven.adoc

```diff
@@ -106,6 +106,23 @@ include::sample[dir="signing/maven-publish/groovy",files="build.gradle[tags=pom-
 include::sample[dir="signing/maven-publish/kotlin",files="build.gradle.kts[tags=pom-customization]"]
 ====
 
+[[publishing_maven:resolved_dependencies]]
+=== Customizing dependencies versions
+
+The set of dependencies and constraints that are added to the POM file will use by default the _declared_ versions from the build file itself.
+Sometimes it is desirable to publish the _resolved_ version instead.
+
+This is done by using the `versionMapping` DSL method which allows to configure the link:{javadocPath}/org/gradle/api/publish/VersionMappingStrategy.html[VersionMappingStrategy]:
+
+.Using resolved versions
+====
+include::sample[dir="signing/maven-publish/groovy",files="build.gradle[tags=versions-resolved]"]
+include::sample[dir="signing/maven-publish/kotlin",files="build.gradle.kts[tags=versions-resolved]"]
+====
+
+In this example above, Gradle will use the versions resolved on the `runtimeClasspath` for dependencies declared in `api` and mapped to Maven `compile`.
```

Comment:
`and mapped to Maven compile.` -> `, which are mapped to Maven's compile scope.`